packages/ses/src/tame-regenerator-runtime.js

```diff
@@ -0,0 +1,31 @@
+import {
+  defineProperty,
+  iteratorPrototype,
+  iteratorSymbol,
+  objectHasOwnProperty,
+} from './commons.js';
+import { accessor, permitted } from './permits.js';
+
+export const tameRegeneratorRuntime = () => {
+  const iter = iteratorPrototype[iteratorSymbol];
+  permitted['%IteratorPrototype%']['@@iterator'] = accessor;
+  defineProperty(iteratorPrototype, iteratorSymbol, {
+    configurable: true,
+    get: () => iter,
+    set(value) {
+      // ignore the assignment on IteratorPrototype
+      if (this === iteratorPrototype) return true;
+      if (objectHasOwnProperty(this, iteratorSymbol)) {
+        this[iteratorSymbol] = value;
+        return true;
```

Comment:
Please double check me, but I don't think a setter's return result is of any significance, so this line should simply be omitted. The proxy set trap is, IIRC, a different story.

```suggestion
```

Reply:
the linter errors if I don't add one

```
 Warning:   15:10  warning  Expected to return a value at the end of method 'set'  consistent-return
```